FrameRate/FrameRate.cpp

```diff
@@ -85,15 +85,24 @@ namespace WPEFramework
             std::lock_guard<std::mutex> guard(m_callMutex);
 
             LOGINFOMETHOD();
+            try{
+            	if (parameters.HasLabel("frequency"))
+            	{
+            		int fpsFrequencyInMilliseconds = DEFAULT_FPS_COLLECTION_TIME_IN_MILLISECONDS;
+	                fpsFrequencyInMilliseconds = std::stod(parameters["frequency"].String());
+			if( fpsFrequencyInMilliseconds > = 100) // make sure min freq is 100 and not less than that.
+			{
+        	    		setCollectionFrequency(fpsFrequencyInMilliseconds);
+            			returnResponse(true);
+			}
+            	}
+		}
+	     catch(...)
+	     {
+		returnResponse(false);
```

Comment:
Pls catch the exception and add some logging and return error indicating that the value is invalid.

Reply:
Added the logs